### PR TITLE
Instrument calls to lhttpc

### PR DESCRIPTION
**Pull request by miriampena**

### CODE REVIEW

File: src/dinerl.erl [hunk 1]
```diff
@@ -389,8 +389,8 @@ batch_write_item(TableName, PutItems, DeleteKeys) ->
     api(batch_write_item,
         [{<<"RequestItems">>,
           [{TableName,
-            lists:map(fun make_batch_put/1, PutItems)
-            ++ lists:map(fun make_batch_delete/1, DeleteKeys)}]}]).
+            lists:map(fun make_batch_put/1, PutItems) ++
```
Comment (elbrujohalcon) [inline src/dinerl.erl:392]: You need to upgrade your formatter plugin.

File: src/dinerl_util.erl [hunk 2]
```diff
@@ -0,0 +1,66 @@
+-module(dinerl_util).
```
Comment (elbrujohalcon) [inline src/dinerl_util.erl:1]: You might want to consider implementing [`erl_stat`](https://github.com/SemanticSugar/erl_utils/blob/master/src/erl_stat.erl) here.

Comment (elbrujohalcon) [reply src/dinerl_util.erl:1]: Or, at least, using its functions instead of reimplementing them.

**Review by elbrujohalcon — CHANGES_REQUESTED** [review]
I think you should just use `erl_stat` for this, like we do in `coinpurse`, instead of adding new callback functions and replicating `erl_stat`'s code in this project.

**Comment by miriampena** [comment]
We cant use erl_stat as this is a public project

File: src/dinerl_util.erl (hunk 2 above)
Comment (miriampena) [reply src/dinerl_util.erl:1]: That was exactly my initial intention but then I realized this is public and erl_stat is not open-sourced just yet :(

Comment (elbrujohalcon) [reply src/dinerl_util.erl:1]: Oh, right! 🤦‍♂️

Comment (elbrujohalcon) [reply src/dinerl_util.erl:1]: Maybe it's worth just bringing `erl_stat` to this project and using it. That way, if we ever make `erl_utils` public, we can just remove the copied module from here and everything keeps working as usual.

**Review by elbrujohalcon — APPROVED** [review]

File: src/dinerl_util.erl (hunk 2 above)
Comment (miriampena) [reply src/dinerl_util.erl:1]: That is a really good suggestion. Ill keep it in mind for the next iteration that is soon to come. Merging to get this deployed.

**Pull request merged**